OrbitSsh/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(OrbitSsh)
+
+# library
+add_library(OrbitSsh STATIC)
+
+target_compile_options(OrbitSsh PRIVATE ${STRICT_COMPILE_FLAGS})
+
+target_compile_features(OrbitSsh PUBLIC cxx_std_17)
+
+target_include_directories(OrbitSsh PUBLIC
+        ${CMAKE_CURRENT_LIST_DIR}/include)
+
+target_sources(OrbitSsh PUBLIC
+        include/OrbitSsh/Channel.h
+        include/OrbitSsh/DirectTcpChannelManager.h
+        include/OrbitSsh/ExecChannelManager.h
+        include/OrbitSsh/LocalSocketManager.h
+        include/OrbitSsh/TunnelManager.h
+        include/OrbitSsh/ResultType.h
+        include/OrbitSsh/Session.h
+        include/OrbitSsh/SessionManager.h
+        include/OrbitSsh/SshManager.h
+        include/OrbitSsh/Socket.h)
```

Comment:
maybe split these so that every commit complies ...

Reply:
done